docker-compose.yml

```
@@ -0,0 +1,20 @@
+version: "3.9"
+
+services:  
+  kinesis:
+    image: "node:16-slim"
```

```suggestion
    image: "node:18-slim"
```

Seems like Node 18 should be here. It's mentioned in [`.nvmrc`](https://github.com/babbel/miza-kinesis/pull/313/files#diff-2c8effdf840690ccb88c7e21e56148978c6adb2c6926530c58ff0e47a9588b44R1).